### PR TITLE
♻ Clearer countdown

### PR DESCRIPTION
**Pull request by staff0rd**
No functional change

### CODE REVIEW

File: elements/reigns/src/Countdown.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    this._value = null;
+  }
+
+  start() {
+    this._value = START;
+  }
+
+  lock() {
+    this._value = LOCK_START;
+    return this;
```
Comment (maxired) [inline elements/reigns/src/Countdown.ts:61]: looks like you added this line so you can write https://github.com/fres-co/fresco-community/pull/92/files#diff-7013a60951a5a76bf49d3d6c17435a605ccc3e5f73433a52cd43c1cca7f147c1R77
if so, then I guess every method (`stop`, `start`, `increment`, `decrement`) should return `this`

**Comment by maxired** [comment]
while testing with 3 participant, I am having an crash.

Screenshot 2022-07-12 at 08 30 44

**Comment by staff0rd** [comment]
@maxired nice catch, will update

File: elements/reigns/src/Countdown.ts (hunk 1 above)
Comment (staff0rd) [reply elements/reigns/src/Countdown.ts:63]: 0b82f01b95c6a39863f7f036a4e949f75f3f7449

**Comment by staff0rd** [comment]
@maxired that crash is fixed

**Pull request merged**